snap/snapcraft.yaml

```diff
@@ -7,7 +7,10 @@ summary: EdgeX Graphical User Interface
 description: Refer to https://snapcraft.io/edgex-ui
 adopt-info: edgex-ui
 
-epoch: 1
+# 0: edinburgh, geneva, hanoi
+# 1: ireland, jakarta, kamakura, levski
```

Comment:
I don't think there was a UI snap up until Jakarta. The epoch was started from 1.
```suggestion
# 1: <=levski
```